velox/exec/fuzzer/AggregationFuzzer.cpp

```diff
@@ -84,6 +84,9 @@ class AggregationFuzzer : public AggregationFuzzerBase {
     // Number of iterations using aggregations over distinct inputs.
     size_t numDistinctInputs{0};
 
+    // Number of iterations using streaming aggregations over distinct inputs.
+    size_t numStreamingDistinctInputs{0};
```

Comment:
This new counter seems to introduce inconsistence. We could streaming aggs with distinct inputs, but not with inputs as is. I suggest to remove this counter for now or figure out how to add it in a consistent way.

Reply:
Got it, removed. I will figure it out and add the related stats separately.